Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -19,24 +19,52 @@ struct GenerateCommand: ParsableCommand, HasTrackableParameters {
     )
     var path: String?
 
+    @Argument(help: """
+    A list of targets in which you'd like to focus. \
```

Comment:
```suggestion
    A list of targets to focus on. \
```

Let's keep the documentation here as brief and to the point as possible.